astroplan/plots/tests/test_sky.py

```diff
@@ -22,3 +22,21 @@ def test_image_example():
     ax = fig.add_subplot(1, 1, 1)
     ax.plot([1, 2, 3])
     return fig
+
+
+@pytest.mark.skipif('not HAS_MATPLOTLIB')
+@pytest.mark.mpl_image_compare
+def test_timezone():
+    import matplotlib.pyplot as plt
+    from astropy import coordinates
+    from astropy import units as u
+    from ..time_dependent import plot_airmass
+    from ... import Observer
+    import datetime
+    import pytz
+
+    bg = coordinates.SkyCoord.from_name('Betelgeuse')
+    vla = Observer(coordinates.EarthLocation.of_site('VLA'))
```

Comment:
Could you please replace VLA with an observatory that's already mocked? See list [here](https://github.com/astropy/astroplan/blob/d5bdec578dccff82461ad62340775a9864aceba8/astroplan/utils.py#L262-L305).